app/templates/events/view/scheduler.hbs

```diff
@@ -10,8 +10,8 @@
         <Input type="text" placeholder={{t "Search"}} @value={{this.filter}} />
         <i class="search icon"></i>
       </div>
-      <h3 class="ui header">{{t 'Unscheduled Sessions'}}</h3>
-      <div class="ui raised segments" style="overflow: auto; height: 750px">
+      <h3 class="ui header">{{t 'Not yet scheduled Sessions'}}</h3>
+      <div class="ui raised segments" style="overflow: auto; height:100%">
```

Comment:
?

Reply:
Sir this was part of issue to change text.